Wireframe/index.html

```diff
@@ -8,26 +8,41 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+      <h1>📄 Wireframe Explorer 📄</h1>
+      <p>a two-dimensional skeletal outline of a webpage or app.</p>
+      <hr />
     </header>
+
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
+        <img src="https://media.geeksforgeeks.org/wp-content/cdn-uploads/20210328185457/How-To-Create-a-Wireframe-in-Software-Design.png" alt="Wireframe design showing page layout" />
+        <h2>WireFrame</h2>
+        <p>A website wireframe, also known as a page schematic or screen blueprint, is a visual guide that represents the skeletal framework of a website.</p>
+        <a href="https://en.wikipedia.org/wiki/Website_wireframe" target="_blank" rel="noopener noreferrer">Read more</a>
       </article>
+
+      <article>
+        <img src="https://gcapes.github.io/git-course/fig/feature-branches.svg" alt="Diagram showing Git feature branches" />
+        <h2>Git Branch</h2>
+        <p>In Git, branching allows developers to create independent lines of development, diverging from the main project line and working on new features or bug fixes without affecting the primary codebase.
+        It's like creating a separate "parallel universe" for your code, enabling parallel development efforts.</p>
+        <a href="..." target="_blank" rel="noopener noreferrer">Learn Git Branching</a>
+      </article>
+
+      <article>
+        <img src="https://i.ytimg.com/vi/jeOfS90Flf8/maxresdefault.jpg" alt="YouTube screenshot explaining README file usage" />
+        <h2>ReadMe</h2>
+        <p>A README file is a plain text file, often used in software repositories or when sharing datasets, that provides a brief introduction and explanation of the project or data.
+        You can add a README file to your repository to tell other people why your project is useful, what they can do with your project, and how they can use it.</p>
+        <a href="" target="_blank" rel="noopener noreferrer">Read more</a>
```

Comment:
@Shayida999 Can you check this link? It should bring the user to an article about readme files.